contracts/BaseStrategy.sol

```diff
@@ -725,7 +722,7 @@ abstract contract BaseStrategy {
         uint256 amountFreed;
         (amountFreed, _loss) = liquidatePosition(_amountNeeded);
         // Send it directly back (NOTE: Using `msg.sender` saves some gas here)
-        want.safeTransfer(msg.sender, amountFreed);
+        SafeERC20.safeTransfer(want, msg.sender, amountFreed);
```

Comment:
This was already using `SafeERC20` under the hood (because of `using SafeERC20 for IERC20;`)

Reply:
I did that because of https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2484

Reply:
Ah okay, yeah this makes sense come to think of it